modules/cf-core/src/methods/state-channel/deposit/controller.ts

```diff
@@ -47,6 +49,17 @@ export default class DepositController extends NodeController {
 
     const channel = await store.getStateChannel(multisigAddress);
 
+    const expectedMultisigAddress = await getCreate2MultisigAddress(
+      channel.userNeuteredExtendedKeys,
+      networkContext.ProxyFactory,
+      networkContext.MinimumViableMultisig,
+      provider
+    );
```

Comment:
you need to use the channels proxy factory address here, not the one from the `networkContext`

`networkContext` could change during the lifetime of the channel, but using the `proxyFactoryAddress` that's stored in the channel should always work.

Reply:
Ahh right that makes sense!